x/txfees/keeper/feetokens_test.go

```diff
@@ -173,10 +173,11 @@ func (suite *KeeperTestSuite) TestFeeTokenConversions() {
 			expectedConvertable: true,
 		},
 		{
-			name:                "unequal value",
-			baseDenomPoolInput:  sdk.NewInt64Coin(baseDenom, 100),
-			feeTokenPoolInput:   sdk.NewInt64Coin("foo", 200),
-			inputFee:            sdk.NewInt64Coin("foo", 10),
+			name:               "unequal value",
+			baseDenomPoolInput: sdk.NewInt64Coin(baseDenom, 100),
+			feeTokenPoolInput:  sdk.NewInt64Coin("foo", 200),
+			inputFee:           sdk.NewInt64Coin("foo", 10),
+			// expected to get 5.000000000005368710 baseDenom without rounding
 			expectedOutput:      sdk.NewInt64Coin(baseDenom, 5),
```

Comment:
oh interesting, this means theres some subtle rounding in the Calc Spot Price function, likely related to the pool weights somehow.

Reply:
(Not an issue of any sort, just relates to the precision we quote reuslts as accurate until. We should probably just truncate precision of the quoted result in SpotPrice in the future)